docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -2,22 +2,81 @@
 
 .. _booting-a-qemu-image:
 
-Booting up a QEMU image
-=======================
+####################################
+Building and booting up a QEMU image
+####################################
+
+Booting a QEMU image can be using one of the two scripts
+
+- ``runqemu``
+- ``qemu_launcher``
+
+Both are scripts but they are offered from different layers. ``runqemu`` is a part of OpenEmbedded while qemu_launcher is a part of meta-pelux. ``qemu_launcher`` requires enabling the virtualization on BIOS while ``runqemu`` is more straight forward and simple. Both options are described below. 
```

Comment:
"a part" -> "part" (in two occurrences)

"straight forward" -> "straightforward"

Also, please break long lines at 80 chars.